packages/d/dpp/xmake.lua

```diff
@@ -78,6 +79,10 @@ package("dpp")
         if package:version():ge("v10.0.23") then
             package:add("defines", "DPP_USE_EXTERNAL_JSON")
         end
+
+        if package:version():ge("v10.0.29") and is_plat("windows") then
+            package:add("cxxflags", "/bigobj /Gy")
```

Comment:
you should pass flags to `tools.xmake` instead of user project.

```lua
local cxflags
if package:is_plat("windows") then
    cxflags  = {"/bigobj", "/Gy"}
end
import("package.tools.xmake").install(package, {}, {cxflags = cxflags})
```

Reply:
This change doesn't seem to have any effect on the build, the "Windows 2019 shared x64" one is still failing because of the error above :thinking: